File: lista_0/fibonac/sol.py
def fibbonacci(n):
    if(n == 1 or n == 2): return 1;
    else: return fibbonacci(n-1) + fibbonacci(n-2);

def fatorial(n):
    fatorial = 1;
    while(n >= 1):
        fatorial *= n
        n -= 1
    return fatorial
    
print(f"10! = {fatorial(10)}")

lista = [fibbonacci(x) for x in range(1, 20)]
print(f"Sequência de Fibonacci: {lista}")

File: lista_0/fibonac/sol.cpp
#include <iostream>

using namespace std;

int fibonacci(int n){
    if(n == 1 | n == 2) return 1;
    return fibonacci(n-1) + fibonacci(n-2);
}

int fatorial(int n){
    int fatorial = 1;
    while (n--){
        if(n == 0) break;
        fatorial = fatorial * n;
    }
    return fatorial;
}

int main(){
    Fibonacci:
    for (int i = 1; i <= 20; i++){
        cout << fibonacci(i) << ", ";
    }

    // Fatorial:
    // cout << "\n" << fatorial(10) << "\n";
    return 0;
}